src/style.css

```
@@ -1 +1,61 @@
-/* Styles go here */
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  --color-link: #00acdc;
+}
+
+.nav__list {
+  list-style: none;
+  padding: 0;
+  margin: 0;
+  display: flex;
+}
+
+.header {
+  padding-left: 50px;
+  padding-right: 50px;
+  height: 60px;
+  text-transform: uppercase;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.logo {
+  margin-right: 20px;
+  display: flex;
+}
+
+.nav_item {
+  position: relative;
+}
+
+.nav_item a {
```

```suggestion
.nav__link {
```
add `nav__link` class instead of using chained selector